src/crypto/crypto_bio.cc

```diff
@@ -271,8 +252,7 @@ size_t NodeBIO::Read(char* out, size_t size) {
   while (bytes_read < expected) {
     CHECK_LE(read_head_->read_pos_, read_head_->write_pos_);
     size_t avail = read_head_->write_pos_ - read_head_->read_pos_;
-    if (avail > left)
-      avail = left;
+    if (avail > left) avail = left;
```

Comment:
I find changes like this less readable and I'm guessing I'm not alone. (It further suggests to me that [we don't actually use `clang-format`](https://github.com/nodejs/node/pull/42668).) I'm sure there's a configuration that could be changed in `.clang-format` if this is a highly-undesriable change.